src/js/view/Listener.js

```diff
@@ -24,20 +38,44 @@ export const onModalClose = () => {
 
 export const onLottoRestart = () => {
   $purchasingAmountInput.value = '';
-  setVisibleResultAreas(false);
+  setVisibleAreas($resultAreas, false);
   onModalClose();
 };
 
-export const onLottosBought = () => {
+export const onPurchasingAmountEntered = () => {
   try {
     const purchasingAmount = $purchasingAmountInput.value;
     validatePurchasingAmount(purchasingAmount);
-    // TODO: 기능 작성 후 역할 분리하기
-    const isPurchasingManually = $checkPurchasingManually.checked;
+    setVisibleAreas($purchasingManuallyForm, true);
+    truncateLottoNumberInput();
+    $autoPurchasingInformationPhrase.innerText = AUTO_N_MESSAGE(purchasingAmount / LOTTO_PRICE);
+  } catch (error) {
+    alert(error.message);
+    console.error(error);
+  }
+};
+
+export const onManualAdd = () => {
```

Comment:
```suggestion
export const handleAddManual = () => {
```
보통 이렇게 표현한답니다!

혹은, 어차피 이 함수가 event를 다루는게 아니기 때문에
```js
export const addManual = () => {
```
이렇게 표현해도 무방할 것 같아요~